docs/version_releases.rst

```diff
@@ -0,0 +1,20 @@
+Version releases
+================
+
+Create new release
+------------------
+
+How create new release you can read `this <https://help.github.com/en/articles/creating-releases>`_.
```

Comment:
I would rephrase as "You can read official Github [instruction] for more info how to create releases."

Reply:
Done.